lib/StagehandPage.ts

```diff
@@ -192,7 +192,7 @@ ${scriptContent} \
     }
 
     const browserbase = new Browserbase({
-      apiKey: process.env.BROWSERBASE_API_KEY,
+      apiKey: process.env.BROWSERBASE_API_KEY || this.stagehand.apiKey,
```

Comment:
Let's prioritize the explicitly defined apikey
```suggestion
      apiKey: this.stagehand.apiKey ?? process.env.BROWSERBASE_API_KEY ,
```